litestar/handlers/http_handlers/base.py

```diff
@@ -285,7 +302,7 @@ def __init__(
         self.response_cookies: Sequence[Cookie] | None = narrow_response_cookies(response_cookies)
         self.response_headers: Sequence[ResponseHeader] | None = narrow_response_headers(response_headers)
 
-        self.sync_to_thread = sync_to_thread
+        self.has_sync_callable = has_sync_callable
```

Comment:
We can remove the class-level annotation for this on line 133 now that we assign it here.